src/components/liveSession/services/liveSession.service.js

```diff
@@ -395,6 +410,33 @@
                 });
             };
 
+            this._getExtendTime = (lessonData) => {
```

Comment:
Remove this func, do this when we start the session and keep the data in the session